app/helpers/encryption_helper.rb

```diff
@@ -0,0 +1,151 @@
+require "csv"
+module EncryptionHelper
+  def assign_secret_key
+    primes = []
+
+    CSV.foreach("primes_test.csv") do |row|
+      primes << row.first.to_i
+    end
+
+    length = primes.length - 1
+
+    key = [primes[rand(length)], primes[rand(length)]]
+
+    while key[0] == key[1]
+      key = [primes[rand(length)], primes[rand(length)]]
+    end
+
+    key
+  end
+
+  def euclidean_algorithm(a, b)
+    euclidean_algorithm = [[a, a/b, b, a%b]]
```

Comment:
Surrounding space missing for operator '/'.<br>Surrounding space missing for operator '%'.